filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -390,13 +390,13 @@ filebeat:
 
 ===== registry_file
 
-The name of the registry file. By default, the registry file is put in the current
-working directory. If the working directory changes for subsequent runs of Filebeat, indexing starts from the beginning again.
+The name of the registry file. If a relative path is used, it is considered relative to the
```

Comment:
I'm not sure what you mean by data path here. Do you mean relative to the folders that are being crawled? Does that mean there is more than one registry file?